config/manager/default-config/deployment.yaml

```diff
@@ -33,6 +33,11 @@ spec:
             defaultMode: 420
             optional: true
             secretName: dynamic-plugins-npmrc
+        - name: dynamic-plugins-registry-auth
+          secret:
+            defaultMode: 256
```

Comment:
Just curious about this. I just tested in my vanilla K8s cluster and, while the file permissions look good (per https://github.com/redhat-developer/rhdh-chart/pull/41#discussion_r1722797257), the problem is that the Secret is mounted as root by K8s. Since the container is running as non-root, the user is not able to read the file, which is owned by `root`. I guess we might run into this issue if the container process tries to access that file, no?

```
$ kubectl exec -it backstage-bs1-645f9b7fd-6tq9m -c install-dynamic-plugins -- /bin/sh 
sh-5.1$ ls -l /opt/app-root/src/.config/containers/auth.json
-r-------- 1 root root 147 Aug 22 15:18 /opt/app-root/src/.config/containers/auth.json
sh-5.1$ cat /opt/app-root/src/.config/containers/auth.json
cat: /opt/app-root/src/.config/containers/auth.json: Permission denied
sh-5.1$
```

Reply:
good catch. i changed the defaultMode to `416` now, which results in`-rw-r----`. Tested it within my cluster and worked.